app/src/pages/inside/stepPage/modals/postIssueModal/postIssueModal.jsx

```diff
@@ -411,9 +421,15 @@ export class PostIssueModal extends Component {
       namedBtsIntegrations,
       intl: { formatMessage },
       data: { items },
+      projectInfo,
     } = this.props;
     const { pluginName, integrationId, fields } = this.state;
     const currentExtension = this.getCurrentExtension();
+    const integrationInfo = {
+      integrationId,
+      projectName: projectInfo.projectName,
```

Comment:
Here we are using projectInfo.projectName but not `activeProjectName || projectInfo.projectName`

Reply:
Post issue form available only from project level, so there is not the case.